src/ORM/Behavior/CounterCacheBehavior.php

```diff
@@ -309,6 +311,12 @@ protected function _getCount(array $config, array $conditions): int
         $config['conditions'] = array_merge($conditions, $config['conditions'] ?? []);
         $query = $this->_table->find($finder, ...$config);
 
-        return $query->count();
+        if (isset($config['useSubQuery']) && $config['useSubQuery'] === false) {
+            return $query->count();
```

Comment:
Why do we need to keep both paths with an option?

Reply:
In case someone's database gets sad from subqueries, but is ok with eventual consistency.

Reply:
Yup, I wanted to provide a way to disable just in case someone had a problem with the subquery usage.